main.go

```diff
@@ -291,22 +291,6 @@ func (s XcodeArchiveStep) ProcessInputs() (Config, error) {
 		}
 	}
 
-	if config.ArtifactName == "" {
```

Comment:
Moved to the Run phase, so it is ran after resolving package deps; this way showBuildSettings would not take as long.